Repo's AE tunnle fix

bartimaeusnek opened the pull request.

repo-alt committed:
GT p2p tunnel was added to AE itself

repo-alt committed:
reverted CoFHCore dependency removal (unrelated to this PR)

repo-alt committed:
Merge pull request #311 from repo-alt/experimental

Half baked EU P2P Tunnel removed, in favour of native implementation

repo-alt committed:
A simple cache for the furnace recipes, to avoid linear iteration
(Should help with https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/6416)

basdxz reviewed (commented):
Looks like a much cleaner implementation, I dislike energy tunneling through AE as a concept tho.

Glease committed:
Fix GTNewHorizons/GT-New-Horizons-Modpack#6586

This makes it waste the extra fuel value, give the optimal output advertised on tooltip and always have an efficiency of 100%.

repo-alt committed:
replace HashMap with a (bounded) Guava Cache

Dream-Master committed:
Merge pull request #312 from repo-alt/experimental

A simple cache for the furnace recipes, to avoid linear iteration

Dream-Master committed:
Merge pull request #313 from GTNewHorizons/patch-lgt-underflow

Fix GTNewHorizons/GT-New-Horizons-Modpack#6586

botn365 committed:
fix horibe tps lag on multie smelter

Dream-Master committed:
Merge pull request #315 from botn365/patch-3

fix horibe tps lag on multie smelter

botn365 committed:
fix energy consumption on OC past tick limit

fix the energy consumption of the large chemical reactor where it would reduce the energy consumption if the it OCd past the tick limit

Dream-Master committed:
Merge pull request #316 from botn365/patch-4

fix LCR energy consumption on OC past tick limit

Dream-Master committed:
Revert "fix energy consumption on OC past tick limit"

This reverts commit efbaa3fcb94da0aea4a4d51519b58df4641e68d9.

botn365 committed:
fix to much power on non perfect OC

botn365 committed:
Merge branch 'experimental' into patch-4

botn365 committed:
put back oldTime

Dream-Master committed:
Merge pull request #317 from botn365/patch-4

fix to much power on non perfect OC

boubou19 committed:
fixed data bank achievement

Dream-Master committed:
Merge pull request #318 from boubou19/experimental

fixed data bank achievement

botn365 committed:
fix furnace recipes with 2+ output

fix multi smelter only given 1 item if the even if the recipe gave more then 1 item

Dream-Master committed:
Merge pull request #319 from botn365/patch-5

fix furnace recipes with 2+ output

bartimaeusnek committed:
Fix AE2 compat to be independant at runtime

Signed-off-by: bartimaeusnek <33183715+bartimaeusnek@users.noreply.github.com>

kaziu687 committed:
Update BaseMetaTileEntity.java

Fixes problem that may cause explosion on Thermos servers with load-chunks-on-request: false

When machine stays at the edge of loaded chunk (where siblings is not loaded) worldObj.getPrecipitationHeight will return -1 (cuz chunk wont be loaded on request). Machine will explode even if there is a roof.

kaziu687 committed:
worldObj.getPrecipitationHeight moved to variable

Dream-Master committed:
Merge pull request #320 from kaziu687/experimental

Update BaseMetaTileEntity.java

botn365 committed:
add QoL for flasks

add the option to set the capacity of flasks trough a assembler and circuit

botn365 committed:
add 720L valua and turn 1000 to circuit 24

botn365 committed:
add 864 and make circuit numbers better

Dream-Master committed:
Merge pull request #321 from GTNewHorizons/QoL-flasks

add QoL for flasks

Dream-Master committed:
fix(GT)add missing NBT Imort

bartimaeusnek committed:
Invalid Muffler Hatch Facing

Added the ability to change the invalid facing orientation on the Muffler Hatch

Signed-off-by: bartimaeusnek <33183715+bartimaeusnek@users.noreply.github.com>

bartimaeusnek committed:
Merge pull request #322 from GTNewHorizons/AE2Independancy

Fix AE2 compat to be independant at runtime

Dream-Master committed:
Merge pull request #323 from GTNewHorizons/hatch_orientation

Cherry-Pick: Invalid Muffler Hatch Facing from GTNH2

botn365 committed:
add overflow voiding to digital chests

botn365 committed:
update lang file

botn365 committed:
speling

botn365 committed:
fix extra } in post tick

botn365 committed:
remove extra if else

Prometheus0000 committed:
Merge pull request #1 from GTNewHorizons/experimental

update

Glease committed:
Make cleanroom workspeed based on height and tier

It is now 225 second per block height. A standard 3x4x3 cleanroom will take 7.5 minute to fire up (down from 8.33 minute). Any wider will not cause it to slowdown. The cleanroom will be overclockable (not perfect).

botn365 committed:
better localisation handeling

Dream-Master committed:
Merge pull request #324 from GTNewHorizons/digital-chest-void-overflow

Digital chest void overflow

Prometheus0000 committed:
Merge pull request #2 from GTNewHorizons/experimental

update

Prometheus0000 committed:
Fix ganymede bee drop

Dream-Master committed:
Merge pull request #325 from Prometheus0000/Bee

#6334 Hopefully fix ganymede bee drop

bartimaeusnek committed:
Merge branch 'experimental' into merged_GTNH1_2
